modules/standard/FileSystem.chpl

```diff
@@ -861,6 +869,10 @@ proc isLink(out error:syserr, name: string): bool {
   extern proc chpl_fs_is_link(ref result:c_int, name: c_string): syserr;
 
   var ret:c_int;
+  var doesExist = exists(error, name);
```

Comment:
You're right that exists gives the wrong value for broken symlinks.  I disagree that we don't want the errors, though.  It seems like hiding them would add to a user's confusion - in the case of EACCES, for instance, the file may or may not exist but the current process can't determine that.  If the user knows the file exists and gets false from isFile, they'll know something is wrong but not *what* is wrong  - providing EACCES will lead them to a solution faster than a "false" return value would.